### PR TITLE
Update unstable Polka-dot JS url (https://polkadot.js.org/apps/)

### PR DESCRIPTION
**Pull request by muddlebee**
Previous : https://polkadot.js.org/apps/
New : https://dotapps.io/ 

Update unstable URL( https://polkadot.js.org/apps/ ) to the stable URL ( https://dotapps.io/ ) as mentioned in the core repository https://github.com/polkadot-js/dotapps.io

### CODE REVIEW

**Comment by muddlebee** [comment]
hey @DrW3RK 👋 

Can you please see if this is okay? People have been reporting this unstability over discord group as well :) 
This works as a long term fix.

**Comment by DrW3RK** [comment]
@muddlebee Thanks for reporting! What unstability are you referring to? https://polkadot.js.org/apps/ should open as expected.

https://dotapps.io/  is an IPFS version that is updated more or less on a weekly basis. Instead of replacing the original link, providing and alternative in the doc might be a better idea!

File: docs/learn/learn-account-generation.md [hunk 1]
```diff
@@ -173,7 +173,7 @@ feasible in your situation.
 
 ### Go to Polkadot-JS Apps
 
-Navigate to [Polkadot-JS Apps](https://polkadot.js.org/apps) and click on "Accounts" underneath the
+Navigate to [Polkadot-JS Apps](https://dotapps.io/) and click on "Accounts" underneath the
```
Comment (profradha) [inline docs/learn/learn-account-generation.md]: Navigate to [Polkadot-JS Apps](https://polkadot.js.org/apps) (or [dotapps.io](https://dotapps.io/)) and click on "Accounts" underneath the

**Review by DrW3RK — CHANGES_REQUESTED** [review]
It does not matter if it is https://polkadot.js.org/apps or https://dotapps.io - both of them point to the same repo. If there is an issue with repo hosting service for https://polkadot.js.org/apps then the IPFS hosted version  https://dotapps.io comes to the rescue.

If https://polkadot.js.org/apps takes too long to load, then the issue might be with the load on the RPC node it is connected to. Simply switching it to a different on the top left dropdown should fix that issue

**Comment by muddlebee** [comment]
hey @DrW3RK 

https://polkadot.js.org/apps/ -> this link never works for me :| and for many others as well at the discord group. How to fix this?

Screenshot : 
Screenshot 2022-02-08 at 8 02 34 PM

**Comment by muddlebee** [comment]
@DrW3RK https://polkadot.js.org/apps/

Sometimes it works. Sometimes it doesn't. Now I am able to open after a few clicks and many have reported. Can we look deeper into this? I can check if you re-direct me to the appropriate resource.

**Comment by muddlebee** [comment]
@DrW3RK anyway I have made the changes you suggested 🚀

**Comment by DrW3RK** [comment]
Thanks @muddlebee 

Yeah, the right place to report this issue is Polkadot JS Apps repo https://github.com/polkadot-js/apps . It is important to do some root cause analysis for the blank screen issue. Check out these two issues and see if either of those is a reason

https://github.com/polkadot-js/apps/issues/6900   (need to update browser to latest version)
https://github.com/polkadot-js/apps/issues/6517     (Get rid of problematic extensions)

For now, I will merge your commits to the Wiki

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**